applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_interface_element.cpp

```diff
@@ -111,15 +112,28 @@ void UPwSmallStrainInterfaceElement<TDim, TNumNodes>::Initialize(const ProcessIn
     const GeometryType::IntegrationPointsArrayType& IntegrationPoints =
         r_geometry.IntegrationPoints(mThisIntegrationMethod);
     const unsigned int NumGPoints = IntegrationPoints.size();
-    const unsigned int VoigtSize  = TDim;
-    if ((mStressVector.size() != NumGPoints) || (mStressVector[0].size() != VoigtSize)) {
+    if (const unsigned int VoigtSize = TDim;
```

Comment:
Let;s avoid repeating the type:
```suggestion
    if (const auto VoigtSize = TDim;
```

Reply:
changed